tests/unit/opflow/gradient/gen_network.py

```diff
@@ -2,115 +2,137 @@
+    output.append("\n")
+    return output
+
 
 def generator(bus_size):
-  output=[]
-  output.append("%% generator data\n")
-  output.append("mpc.gen = [\n")
-  for i in range(3, int(bus_size) * 4, 4):
-      output.append(f"{i} 10.00 0.00 9900.00 -9900.00 1.0000 100.00 1 10.00 10.00 0.00 0.00 0.00 0.00 0.00 0.00 0 0 0 010.0000\n")
-  output.append("];\n")
-  output.append("\n")
-  return output
+    output = []
+    output.append("%% generator data\n")
+    output.append("mpc.gen = [\n")
+    for i in range(3, int(bus_size) * 4, 4):
+        output.append(
+            f"{i} 10.00 0.00 9900.00 -9900.00 1.0000 100.00 1 10.00 10.00 0.00 0.00 0.00 0.00 0.00 0.00 0 0 0 010.0000\n")
+    output.append("];\n")
+    output.append("\n")
+    return output
+
 
 def gen_cost(bus_size):
-  output=[]
-  output.append("%% generator cost data\n")
-  output.append("mpc.gencost = [\n")
-  for _ in range(int(bus_size)):
-    output.append("2 0 0 4 0.0000 0.450 0.100 8.00\n")
-  output.append("];\n")
-  output.append("\n")
-  return output
+    output = []
+    output.append("%% generator cost data\n")
+    output.append("mpc.gencost = [\n")
+    for _ in range(int(bus_size)):
+        output.append("2 0 0 4 0.0000 0.450 0.100 8.00\n")
+    output.append("];\n")
+    output.append("\n")
+    return output
+
 
 def branch(bus_size):
-  output=[]
-  output.append("%% branch data\n")
-  output.append("mpc.branch = [\n")
-  for i in range(int(bus_size)):
-    bus_idx = (4 * i + 1)
-    output.append(f"{bus_idx} {bus_idx + 1} 0.000000 0.000010 0.00000 0.00 0.00 0.00 0.00000 0.000 1 0.00 0.00 0.00 0.00 0.00 0.00\n")
-    output.append(f"{bus_idx + 1} {bus_idx + 2} 0.000000 0.000010 0.00000 0.00 0.00 0.00 1.00000 0.000 1 0.00 0.00 -10.00 -0.00 10.00 0.00\n")
-    output.append(f"{bus_idx + 1} {bus_idx + 3} 0.000000 0.000010 0.00000 0.00 0.00 0.00 0.00000 0.000 1 0.00 0.00 10.00 10.00 -10.00 -10.00\n")
-    output.append(f"{bus_idx + 3} {bus_idx + 4} 0.000000 0.000010 0.00000 0.00 0.00 0.00 0.00000 0.000 1 0.00 0.00 0.00 0.00 0.00 0.00\n")
-  output.append("];\n")
-  output.append("\n")
-  return output
+    output = []
+    output.append("%% branch data\n")
+    output.append("mpc.branch = [\n")
+    for i in range(int(bus_size)):
+        bus_idx = (4 * i + 1)
+        output.append(
+            f"{bus_idx} {bus_idx + 1} 0.000000 0.000010 0.00000 0.00 0.00 0.00 0.00000 0.000 1 0.00 0.00 0.00 0.00 0.00 0.00\n")
+        output.append(
+            f"{bus_idx + 1} {bus_idx + 2} 0.000000 0.000010 0.00000 0.00 0.00 0.00 1.00000 0.000 1 0.00 0.00 -10.00 -0.00 10.00 0.00\n")
+        output.append(
+            f"{bus_idx + 1} {bus_idx + 3} 0.000000 0.000010 0.00000 0.00 0.00 0.00 0.00000 0.000 1 0.00 0.00 10.00 10.00 -10.00 -10.00\n")
+        output.append(
+            f"{bus_idx + 3} {bus_idx + 4} 0.000000 0.000010 0.00000 0.00 0.00 0.00 0.00000 0.000 1 0.00 0.00 0.00 0.00 0.00 0.00\n")
+    output.append("];\n")
+    output.append("\n")
+    return output
```

Comment:
These are all just formatting changes right? And precommit makes these changes itself upon push?

Reply:
Yeah all formatting changes. These files weren't linted at all before so that's why these are being touched for the first time.

pre-commit runs a check when you run `git commit`, and then applies changes to files as needed. You then re-add them, run `git commit` again, and pre-commit checks will be a noop and your commit will go through just fine.